### ISSUE TITLE
E3001 error - AWS::ServiceCatalog::Portfolio

### ISSUE THREAD

**Issue by whoDoneItAgain** [aws-cloudformation/cfn-lint#2866]
### CloudFormation Lint Version

0.79.9

### What operating system are you using?

windows

### Describe the bug

AWS::ServiceCatalog::Portfolio is being reported as not supported in us-west-2 since 0.79.9. does report as supported in 0.79.8

deployed a test stack in us-west-2 and it deployed successfully.

### Expected behavior

resource type doesnt generate an error.

### Reproduction template

```
AWSTemplateFormatVersion: 2010-09-09
Description: AWS Service Catalog Portfolio

Resources:
  # Service Catalog Portfolio
  SCPortfolio:
    Type: AWS::ServiceCatalog::Portfolio
    Properties:
      AcceptLanguage: en
      Description: 'Test - portfolio'
      DisplayName: 'Test - portfolio'
      ProviderName: Test

```

**Comment by whoDoneItAgain** [comment]
did a little research and it might be an upstream issue with the cfn spec for us-west-2:

138.0.0 spec snippet:
![image](https://github.com/aws-cloudformation/cfn-lint/assets/12127434/83240dc2-5890-49ee-a448-bc86b79fa987)

137.0.0 spec snippet:
![image](https://github.com/aws-cloudformation/cfn-lint/assets/12127434/2fe3d5bf-e8b8-4200-aa50-65d9d93d62cd)

**Comment by kddejong** [comment]
Looking into it

**Comment by kddejong** [comment]
There is a corresponding internal issue for this one.  I'll be tracking it and will update this issue as I can.

**Comment by whoDoneItAgain** [comment]
sounds good. thanks @kddejong

**Comment by kddejong** [comment]
The new version of the spec resolves this.  Will get this released on Monday but for now you should be able to do `cfn-lint --update-specs`

**Comment by whoDoneItAgain** [comment]
confirmed fixed.
